src-gui/meshpeer.h

```diff
@@ -18,11 +18,11 @@ class MeshPeer : public QObject
 
     enum COMMANDSTATUS {
         none = 0,
-        sended,			//wyslano do noda galaxy
-        recived,		//otrzymano potwierdzenie z sieci galaxy
-        acknowled,		//zsynchronizowano w sieci galaxy
-        timeout,		// timeout komendy
-        fail			//nieudane wykonnie komendy na no
+        sended,
```

Comment:
commends would be nice here (in EN)

// sent to node
// got confirmation

etc